src/ImGui.NET.SampleProgram/Program.cs

```diff
@@ -207,6 +209,20 @@ private static unsafe void SubmitUI()
                 ImGui.Text("Memory editor currently supported.");
                 // _memoryEditor.Draw("Memory Editor", _memoryEditorData, _memoryEditorData.Length);
             }
+            
+            // ReadOnlySpan<char> and .NET Standard 2.0 tests
+            TestStringParameterOnDotNetStandard.Text(); // String overloads should always be available.
+            
+            // On .NET Standard 2.1 or greater, you can use ReadOnlySpan<char> instead of string to prevent allocations.
+            long allocBytesStringStart = GC.GetAllocatedBytesForCurrentThread();
+            ImGui.Text($"Hello, world {Random.Shared.Next(100)}!");
+            long allocBytesStringEnd = GC.GetAllocatedBytesForCurrentThread() - allocBytesStringStart;
+            Console.WriteLine("GC (string): " + allocBytesStringEnd);
+                
+            long allocBytesSpanStart = GC.GetAllocatedBytesForCurrentThread();
+            ImGui.Text($"Hello, world {Random.Shared.Next(100)}!"); // Note that this call will STILL allocate memory due to string interpolation, but you can prevent that from happening by using an InterpolatedStringHandler.
```

Comment:
Isn't this same as the block above?

Reply:
Yeah, I think I accidentally removed `AsSpan()`.

Reply:
As far as I'm aware that has always been the only option.